changelog.md

```diff
@@ -272,6 +272,8 @@ proc mydiv(a, b): int {.raises: [].} =
   an experimental feature. In other words, you don't have to write pragma
   `{.experimental: "forLoopMacros".}` if you want to use them.
 
+- Tuple expressions are now parsed consistently as `nnkTupleConstr` nodes.
```

Comment:
changes like this require either bumping NimPatch (and nimVersionCT, see https://github.com/nim-lang/Nim/pull/14648) or adding this to condsyms.nim:
```nim
defineSymbol("nimHasUnifiedTuple")
```

so that code can be written with fwd and bwd compatibility when needed